lib/reporters/TestReporter.js

```diff
@@ -0,0 +1,142 @@
+import {Test, Suite} from "../Data.js";
```

Comment:
>  this can be used to test if an external test framework implements the JsReporters standard correctly

That sounds good to me. I'm not really sure how that would work though. I guess we can leave it here for now, then try to apply this to a runner, e.g. PR against Mocha that includes the test.